### DISCUSSION TITLE
MMDet 新 config 体验与适配

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#357] [MMDetection]
# MMDet 新 config 体验与适配 

根据[Config适配文档]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc) )内容，测试 MMDet 中的新 config：
1. 选择 configs/panoptic_fpn/panoptic-fpn_r50_fpn_1x_coco.py 配置适配，首先需要确保新配置测试功能正常，有条件的需要简单验证训练过程是否符合预期
2. 尝试修复并汇报过程中遇到的任何问题，以及给出改进意见

## Task Information

**Repository**：MMDetection

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：Crescent-Saturn

**Claim Time**：2023年10月27日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/11109

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Crescent-Saturn** [comment]
Claim the mission

**Reply by Crescent-Saturn** [reply]
PR link: [[MMSIG#357] Add new configs for panoptic_fpn](https://github.com/open-mmlab/mmdetection/pull/11109)
